test/f90_correct/src/implied_do1.f90

```diff
@@ -0,0 +1,60 @@
+!
+! Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+! See https://llvm.org/LICENSE.txt for license information.
+! SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+!
+! Test for implied-do containing function call
+
+program p
+  implicit none
+  integer :: i, j
+  integer :: cmp1(12) =  [5, 10, 1, 5, 10, 2, 2, 5, 10, 3, 3, 3]
+  integer :: cmp2(12) =  [5, 10, 1, 1, 5, 10, 2, 2, 5, 10, 3, 3]
+  integer :: cmp3(17) = [3, 4, 1, 3, 4, 2, 2, 3, 4, 2, 2, 3, 4, 4, 4, 4, 4]
+ 
+  if (size(cmp1) /= size((/ ((/ 5, 10, gen (i) /), i = 1, 3) /))) STOP 1
+  if (any(shape(cmp1) /= shape((/ ((/ 5, 10, gen (i) /), i = 1, 3) /)))) STOP 2
+  if (any(lbound(cmp1) /= lbound((/ ((/ 5, 10, gen (i) /), i = 1, 3) /)))) &
+    STOP 3
```

Comment:
Could you make the code appear more consistent? Either put every `IF ... STOP` statement on a single line, or always break it into two lines with the line continuation character `&`. The inconsistency in the code now is jarring.

Reply:
Thanks.